shopfloor/models/shopfloor_menu.py

```diff
@@ -140,6 +140,8 @@ class ShopfloorMenu(models.Model):
     no_prefill_qty_is_possible = fields.Boolean(
         compute="_compute_no_prefill_qty_is_possible"
     )
+    allow_rma_is_possible = fields.Boolean(compute="_compute_allow_rma_is_possible")
+    allow_rma = fields.Boolean(string="Allow create returns", default=False, help="")
```

Comment:
rename rma by return. An RMA is an authorization to return.
```suggestion
    allow_create_return_is_possible = fields.Boolean(compute="_compute_allow_rma_is_possible")
    allow_create_return = fields.Boolean(string="Allow create returns", default=False, help="")
```